client/src/components/container/Root.js

```diff
@@ -34,6 +35,7 @@ class Root extends Component {
             <Route exact path={HOME} component={Home} />
             <Route exact path={ABOUT_US} component={AboutUs} />
             <Route exact path={USER_PAGE} component={UserPage} />
+            <Route exact path={STATS} component={StatsPage} />
```

Comment:
I would make this a different link than STATS.

You should make two links.
STATS = "/stats"
STATS_SERVLET = "/api/stats"

Here you should use STATS because this doesn't invoke the servlet directly. This will also make the url in your url bar localhost:3000/stats rather than localhost:3000/api/stats

Ideally we want to keep the client and server links explicitly clear. Fetches should always use servlet urls (/api/) and everything else should not have /api/